jsapp/js/components/modalForms/bulkEditSubmissionsForm.es6

```diff
@@ -575,23 +550,18 @@ class BulkEditRowForm extends React.Component {
           </bem.FormView__cell>
         </bem.FormView__cell>
 
-        <bem.SimpleTable m='bulk-edit-responses'>
-          <bem.SimpleTable__header>
-            <bem.SimpleTable__row>
-              <bem.SimpleTable__cell>{t('Response value')}</bem.SimpleTable__cell>
-
-              <bem.SimpleTable__cell>{t('Frequency')}</bem.SimpleTable__cell>
-
-              <bem.SimpleTable__cell>{t('Percentage')}</bem.SimpleTable__cell>
-
-              <bem.SimpleTable__cell>{t('Action')}</bem.SimpleTable__cell>
-            </bem.SimpleTable__row>
-          </bem.SimpleTable__header>
-
-          <bem.SimpleTable__body>
-            {Array.from(this.getUniqueResponses()).map(this.renderResponseRow)}
-          </bem.SimpleTable__body>
-        </bem.SimpleTable>
+        <Box mt='lg'>
```

Comment:
I'd prefer that `mt='lg'` would be applicable to `SimpleTable` itself. See suggestion over there